src/tests/conftest.py

```
@@ -24,7 +21,9 @@ async def reset_db():
 @pytest_asyncio.fixture(scope="function")
 async def client():
     """Provide an asynchronous test client for making HTTP requests."""
-    async with AsyncClient(transport=ASGITransport(app=app), base_url="http://test") as async_client:
+    async with AsyncClient(
+        transport=ASGITransport(app=app), base_url="http://test"
+    ) as async_client:
```

AsyncClient usage: creating the client with `ASGITransport(app=app)` and `base_url="http://test"` is valid. Two small suggestions:
- Consider adding a short `timeout` so tests fail fast if the app stalls.
- Alternatively you can pass `app=app` directly to `AsyncClient` (it will use an ASGI transport internally). Both approaches work; keep whichever is consistent across tests in the repo.